src/richie/apps/core/templates/social-networks/blogpost-badges.html

```diff
@@ -10,6 +10,7 @@
             title="{% trans "Share on Facebook" %}"
         >
             <svg role="img">
+                <title>{% trans "Facebook" %}</title>
```

Comment:
By doing this, screen readers might announce both the anchor title and the svg title, or just one or the other.

I think the best, for svg icon links like this one and those below, would be to:

- remove the `title` attribute from the wrapping `<a>`
- have the svg `title` tag convey the meaning of the anchor, not the image by itself. So, "Share on Facebook" here.
- add an `aria-label` on the svg tag with the exact same content as the svg `title` tag. This is mostly for people using old screen readers that do not understand correctly the `title` tag.

This should make screen readers announce stuff only one time, plus keep the mouse tooltip. What do you think?

Reply:
Sounds great. Will do that.